exercises/accumulate/Tests/accumulateTests/accumulateTests.swift

```diff
@@ -21,10 +19,8 @@ private extension XCTest {
     }
 }
 
-class AccumulateTest: XCTestCase {
-
+class accumulateTests: XCTestCase {
```

Comment:
Only the main folder per excersice needs to keep its name so the script can find and serve on command line.